zk-token-sdk/src/zk_token_proof_instruction.rs

```diff
@@ -132,6 +132,24 @@ pub enum ProofInstruction {
     ///   `PubkeyValidityData`
     ///
     VerifyPubkeyValidity,
+
+    /// Verify a 64-bit range proof.
+    ///
+    /// This instruction can be configured to optionally create a proof context state account.
+    ///
+    /// Accounts expected by this instruction:
+    ///
+    ///   * Creating a proof context account
+    ///   0. `[writable]` The proof context account
```

Comment:
This is common for all the other instructions, but I think it'd be helpful to also say that the "The proof context account" is expected to be allocated to the expected data size and assigned to the proof program before invoking this instruction

Reply:
Yes, you are right. I'll include that remark in the instruction description.